sdk/cognitiveservices/azopenai/autorest.md

```diff
@@ -9,7 +9,7 @@ input-file:
 
 output-folder: ../azopenai
 clear-output-folder: false
-module: github.com/Azure/azure-sdk-for-go/sdk/cognitiveservices/azopenai
+module: github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai
```

Comment:
Is this more `git mv` shenanigans or does this need to be copied?

Reply:
Accidental search and replace. :|